packages/playground/applet/package.json

```diff
@@ -21,6 +22,7 @@
     "sass": "^1.74.1",
     "serve": "^14.2.1",
     "typescript": "^5.4.4",
+    "unocss": "^0.59.0",
```

Comment:
We have unocss in the root package.json (https://github.com/vuejs/devtools-next/blob/main/package.json#L106), so we needn't add it again in here.